content/css/concepts/background/terms/background-clip/background-clip.md

```diff
@@ -73,3 +73,7 @@ body {
   color: transparent;
 }
 ```
+
+## Illustration of Background-Clip
+
+![Rendered background-clip tag](https://raw.githubusercontent.com/Codecademy/docs/main/media/rendered-background-clip.png)
```

Comment:
It looks like the image does not exactly match the properties of the examples. The image below uses a double border, but the examples specify a single dashed line. Also the numbers do not match the examples, content-box should be number 1. Please update the image to match